src/python/dxpy/scripts/dx.py

```diff
@@ -2989,6 +2989,15 @@ def run_body(args, executable, dest_proj, dest_path, preset_inputs=None, input_n
         "extra_args": args.extra_args
     }
 
+    if any([args.watch or args.ssh or args.allow_ssh]):
+        if run_kwargs["priority"] in ["low", "normal"]:
+            print(fill(BOLD("WARNING") + ": You have requested that jobs be run under " +
+                    BOLD(run_kwargs["priority"]) +
+                    " priority, which may cause them to be restarted at any point, interrupting interactive work."))
+            print()
+        else:
+            run_kwargs["priority"] = "high"
+    
     if run_kwargs["priority"] in ["low", "normal"] and not args.brief:
```

Comment:
Shall we suppress the warning when `--brief` just like below? @commandlinegirl @emiloslavsky

Reply:
Yes, imo it should not be printed with --brief, which will be consistent with out other commands. With this flag we usually only print ID or a short string in json or nothing.

Reply:
agree

Reply:
Great I will merge the check with the one below. Thanks!